slack_bolt/oauth/internals.py

```diff
@@ -126,3 +129,20 @@ def select_consistent_installation_store(
     else:
         # only oauth_flow_store is available
         return oauth_flow_store
+
+
+def build_detailed_error(reason: str) -> str:
+    if reason == "invalid_browser":
+        return (
+            f"{reason}: Perhaps, you reloaded this page or "
+            "didn't start the OAuth flow from the valid starting URL. "
+            "Another possibility is that your /slack/install URL might not be https:// URL."
```

Comment:
```suggestion
            f"{reason}: This can occur due to page reload, "
            "not beginning the OAuth flow from the valid starting URL, or "
            "the /slack/install URL not using https://"
```